classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_reader :students
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] To implement the belongs side of the relationship between student and classroom, a method that serves as the 'belongs-to' side (a student belongs to a classroom) should be present in the Student class, something like so: 

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```